src/main/java/org/jboss/xavier/integrations/migrationanalytics/business/AbstractVMWorkloadInventoryCalculator.java

```diff
@@ -100,11 +104,16 @@
     protected <T> List<T> readListValuesFromExpandedEnvVarPath(String envVarPath, Map vmStructMap) {
         String expandParamsInPath = getExpandedPath(envVarPath, vmStructMap);
```

Comment:
A very minor point but...Just for readability and clarity, it might be a good idea to rename this local variable so it's not the same as the method 15-20 lines further down in the class.

Reply:
+1